k8s/production/spack/gantry-spack-io/ingress.yaml

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: gantry-ingress
+  namespace: spack
+
+spec:
+  ingressClassName: nginx
+  tls:
+  - secretName: tls-gantry
+  rules:
+  - host: gantry.spack.io
+    http:
+      paths:
+      - pathType: ImplementationSpecific
+        path: /gantry
+        backend:
+          service:
+            name: gantry-service
+            port:
+              number: 80
```

Comment:
Is it possible to avoid exposing this service to the public internet, i.e. just run it within the k8s internal network where any gitlab runner pods would be able to access it?

Reply:
I think so (I removed it for now), but I'll double check with other people before completely getting rid of it